index.html

```diff
@@ -19,161 +19,216 @@
           src="images/graphics/image_geometry_1.png"
           alt="logo"
         />
+        <ul class="navigation-nav">
+          <li class="nav-item active">
+            <a class="nav-link" href="#landing">Hello</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#portfolio">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#contact">Contact</a>
+          </li>
+        </ul>
         <div class="toggle">
           <div class="line"></div>
           <div class="line"></div>
           <div class="line"></div>
         </div>
       </nav>
       <main>
-        <section class="landing">
-          <h1 class="title">Teshome Kurabachew</h1>
-          <div class="about">
-            <p>
-              Hello! I am a software developer! I can help you build a product,
-              feature or website. Taka a look of my works. If you like what you
-              see and have a project you need coded, don’t hesitate and contact
-              me.
-            </p>
-            <button class="btn" type="button">Start Collaboration</button>
-          </div>
-          <div class="img-geometry">
-            <img
-              class="img-geometry-1"
-              src="images/graphics/image_geometry_2.png"
-              alt="Graphic picture"
-            />
-          </div>
-        </section>
-        <section class="work">
-          <h2 class="title">Projects</h2>
-          <div class="proj-card">
-            <div class="proj-img">
-              <img src="images/project/Rectangle21.png" alt="Project Picture" />
-            </div>
-            <div class="project-detail">
-              <h2 class="project-name">Project name goes here</h2>
-              <ul class="languages">
-                <li>HTML/CSS</li>
-                <li>Ruby on Rails</li>
-                <li>JavaScript</li>
-              </ul>
-              <a href="" class="btn btn-pro"
-                >See this project <span class="arrow">&rarr;</span></a
-              >
+        <section class="landing" id="landing">
+          <div class="landing-element landing-item">
+            <div class="landing-title">
+              <h1 class="title">Teshome Kurabachew</h1>
```

Comment:
- [ ] OPTIONAL:  I recommend giving some space between `landing` and `work` sections. 

Figma design  | Your design
------------- | -------------
<img width="552" alt="Screenshot 2022-02-18 at 13 24 29" src="https://user-images.githubusercontent.com/77038610/154654851-eaceadf2-96e3-4e1a-a011-7b0493385195.png">  | <img width="741" alt="Screenshot 2022-02-18 at 13 24 02" src="https://user-images.githubusercontent.com/77038610/154654784-656d73b6-cc1f-41d0-bcdb-31c2be03ad61.png">